README.md

```diff
@@ -1,2 +1,28 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a): The 'HasDuplicate(list)' function goes compares every member of a list to every member of the same list (including, rather inefficiently, comparing the same member against itself). 
+If i and j are the same, it also asks if it is the same space in the list. If it isn't, the list contains a duplicate entry and we return true.
+
+(b): The running time of the algorithm is quadratic ( O(n^2) ) because, since the algorithm is comparing each element of the list (including the same element), it is going through the the list exactly n^2 times.
```

Comment:
Not quite true that it goes through the list n^2 times -- it goes through n times, each time going through all n elements, hence n*n or n^2.